modules/cert-manager-install-cli.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-install-cli_{context}"]
+= Installing the {cert-manager-operator} by using the CLI
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a new project named `cert-manager-operator` by running the following command:
++
+[source, terminal]
+----
+oc new-project cert-manager-operator
+----
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file, for example, `operatorGroup.yaml`, with the following content:
++
+[source, yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  targetNamespaces:
+  - "cert-manager-operator"
+----
+
+.. Create the `OperatorGroup` object by running the following command:
++
+[source, terminal]
+----
+oc create -f operatorGroup.yaml
+----
+
+. Create a `Subscription` object:
+
+.. Create a YAML file, for example, `subscription.yaml`, that defines the `Subscription` object:
++
+[source, yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  channel: stable-v1
+  name: openshift-cert-manager-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic
+  startingCSV: cert-manager-operator.v1.13.0
+----
+
+.. Create the `Subscription` object by running the following command:
+[source, terminal]
+----
+oc create -f subscription.yaml
+----
+
+.Verification
+
+. Verify that the OLM subscription is created by running the following command:
++
+[source, terminal]
+----
+oc get subscription -n cert-manager-operator
+----
++
+.Example output
+[source, terminal]
+----
+NAME                              PACKAGE                           SOURCE             CHANNEL
+openshift-cert-manager-operator   openshift-cert-manager-operator   redhat-operators   stable-v1
+----
+
+. Verify whether the Operator is successfully installed by running the following command:
++
+[source, terminal]
+----
+oc get csv -n cert-manager-operator
```

Comment:
The rest of the commands in the "Verification" section are missing the `$` in the command. 
```suggestion
$ oc get csv -n cert-manager-operator
```
Adding the comment here so I'm not repeating